backend/tracim_backend/tests/functional/test_workspaces.py

```diff
@@ -3397,7 +3403,9 @@ def test_api__post_content_create_generic_content__ok_200__in_folder(self) -> No
         }
         # INFO - G.M - 2018-06-165 - Verify if new content is correctly created
         active_contents = self.testapp.get('/api/v2/workspaces/1/contents', params=params_active, status=200).json_body  # nopep8
-        assert res.json_body in active_contents
+        assert res.json_body['content_id'] in [
+            content['content_id'] for content in active_contents
+        ]
```

Comment:
by the way @inkhey you should split your tests... 3500 lines of code in one file... which includes tests about user invitation and content creation... this is not logical.

Please create an issue in order to split tests - and more generally to rationalize test code.

Reply:
see https://github.com/tracim/tracim/issues/1081